tests/phpunit/integration/tests/Admin/Cross_Origin_Isolation.php

```diff
@@ -17,141 +17,142 @@
 
 namespace Google\Web_Stories\Tests\Integration\Admin;
 
-use Google\Web_Stories\Tests\Integration\TestCase;
+use Google\Web_Stories\Tests\Integration\DependencyInjectedTestCase;
 
 /**
  * @coversDefaultClass \Google\Web_Stories\Admin\Cross_Origin_Isolation
  */
-class Cross_Origin_Isolation extends TestCase {
+class Cross_Origin_Isolation extends DependencyInjectedTestCase {
 	/**
 	 * Admin user for test.
 	 *
 	 * @var int
 	 */
-	protected static $admin_id;
+	private $admin_id;
 
 	/**
 	 * Contributor user for test.
 	 *
 	 * @var int
 	 */
-	protected static $contributor_id;
+	private $contributor_id;
 
-	public static function wpSetUpBeforeClass( $factory ) {
-		self::$admin_id = $factory->user->create(
+	/**
+	 * Test instance.
+	 *
+	 * @var \Google\Web_Stories\Admin\Cross_Origin_Isolation
+	 */
+	private $instance;
+
+	public function set_up() {
+		parent::set_up();
+
+		// Deliberately NOT created in wpSetUpBeforeClass() because this class contains running
+		// in separate processes, which means tearDownAfterClass() (which deletes all WP data)
+		// is run multiple times, causing the story not to be available anymore.
+		$this->admin_id = self::factory()->user->create(
 			[ 'role' => 'administrator' ]
 		);
 
-		self::$contributor_id = $factory->user->create(
+		$this->contributor_id = self::factory()->user->create(
 			[ 'role' => 'contributor' ]
 		);
+
+		$this->instance = $this->injector->make( \Google\Web_Stories\Admin\Cross_Origin_Isolation::class );
 	}
 
-	public function set_up() {
-		parent::set_up();
+	public function tear_down() {
+		delete_user_meta( $this->admin_id, \Google\Web_Stories\User\Preferences::MEDIA_OPTIMIZATION_META_KEY );
+		delete_user_meta( $this->contributor_id, \Google\Web_Stories\User\Preferences::MEDIA_OPTIMIZATION_META_KEY );
```

Comment:
```suggestion
		delete_user_meta( $this->contributor_id, \Google\Web_Stories\User\Preferences::MEDIA_OPTIMIZATION_META_KEY );
		delete_user_meta( $this->contributor_id, \Google\Web_Stories\User\Preferences::ONBOARDING_META_KEY );
```

Reply:
Why would this be needed? Neither the `Cross_Origin_Isolation` class nor this test use this user meta